src/styles/typing.css

```diff
@@ -0,0 +1,78 @@
+.sc-typing {
+  padding-top: 20px;
+  width: 300px;
+  margin: auto;
+}
+
+.sc-typing--circle {
```

Comment:
Is `.sc-typing--circle` the best name here? The dots are also circles, and to be honest the message that contains them is more of an oval or a rounded rect. I'd call this `.sc-typing--message` or `.sc-typing--body`

Reply:
ok